components/TextFields/src/MDCTextInputControllerDefault.m

```diff
@@ -972,6 +952,44 @@ + (void)setInlinePlaceholderColorDefault:(UIColor *)inlinePlaceholderColorDefaul
                                        : MDCTextInputDefaultInlinePlaceholderTextColorDefault();
 }
 
+- (UIFont *)inlinePlaceholderFont {
+  return _inlinePlaceholderFont ?: [self class].inlinePlaceholderFontDefault;
+}
+
+- (void)setInlinePlaceholderFont:(UIFont *)inlinePlaceholderFont {
+  if (![_inlinePlaceholderFont isEqual:inlinePlaceholderFont]) {
+    _inlinePlaceholderFont = inlinePlaceholderFont;
```

Comment:
We are letting the iVar go to nil and assume we will always use the getter?

style guide discourages accessors in some places.
http://google.github.io/styleguide/objcguide.html#avoid-accessors-during-init-and-dealloc

Reply:
I guess we are already not following the guidance is this component. We should consider doing so.

Reply:
This will be handled in a future PR.